script/Deploy_1.s.sol

```diff
@@ -10,12 +10,14 @@ import {OrallyMulticall} from "src/OrallyMulticall.sol";
 import {ApolloCoordinatorV2} from "src/apollo/ApolloCoordinatorV2.sol";
 import {OrallyPriceFeed} from "src/examples/OrallyPriceFeed.sol";
 
+import {ApolloCoordinatorV2} from "src/apollo/ApolloCoordinatorV2.sol";
+
 contract Deploy_1 is Script {
     address constant pmaAddress = 0x05C3F2A3Ae0b7f3775044EEFED8a864c47125F19;
     address constant sybilAddress = 0x60825063CB0f4EF508854Ad4913f3a6de86B3807;
 
     // different between chains, could be added in different deployment script () on already deployed infrastructure
-    address constant amaAddress;
+    address constant amaAddress = 0xA40a0617BA2b446b0300eeD8B7D8A4C790216314;
```

Comment:
I removed concrete address here for reason, do not to forget to change on correct address of chain during deploy

Reply:
forge doesn't compile anything if there any errors in any contract  :(